### PR TITLE
add currency #176

### LINKED ISSUE #176: Add sterling, dollars (any other) currencies select tag in Form/GetDetails FE

### PR DESCRIPTION
**Pull request by amanshawar**
add currency to price fix #176

### CODE REVIEW

**Review by FidaaHersh — APPROVED** [review]

**Review by orjwan98 — APPROVED** [review]

File: client/src/components/Shared/Form/index.js [hunk 1]
```diff
@@ -199,13 +202,26 @@ class Form extends Component {
...
             </StyledSelect>
 
-            <StyledInput
-              onChange={toggleOpen}
-              type="text"
-              name="price"
-              value={selected_price}
-              placeholder="price"
-            />
+            <StyledPriceConatiner>
```
Comment (jema28) [inline client/src/components/Shared/Form/index.js]: Spelling error here.

File: client/src/components/Shared/Form/form.style.js [hunk 2]
```diff
@@ -80,15 +80,34 @@ export const StyledSelect = styled.select`
...
+	}
+`;
+
+
 export const StyledOption = styled.option`
 	color: #4C5268;
 `;
 
+export const StyledPriceConatiner = styled.div`
+display: flex;
```
Comment (jema28) [inline client/src/components/Shared/Form/form.style.js]: css indentation pleaaaase!

File: client/src/components/Shared/Form/form.style.js [hunk 3]
```diff
@@ -80,15 +80,34 @@ export const StyledSelect = styled.select`
 	}
 `;
 
+export const StyledSelectCurrency = styled.select`
+  width:60px;
+  max-width:60px;
+	background-color: #FBFBFB;
```
Comment (jema28) [inline client/src/components/Shared/Form/form.style.js]: Too much indentation here?

**Review by jema28 — CHANGES_REQUESTED** [review]

**Comment by jema28** [comment]
Really nice job! 😍Default to pounds, fix indentations and then ready to merge 👍

**Review by jema28 — APPROVED** [review]
👍

**Review by Hadeel-Salamin — APPROVED** [review]

**Pull request merged**